README.md

```diff
@@ -45,15 +33,47 @@ COMPOSE_DOCKER_CLI_BUILD=1 docker-compose build
 docker-compose up
 ```
 
-The server application will be available on the port you configured with `DOCKER_LDES_SERVER_PORT` (or port 8080 by default).
-
-#### IntelliJ
+### Maven
```

Comment:
It would be good to extend this with:
* build locally
* run unit tests with and without coverage
* run integration tests with and without coverage
* run all tests with and without coverage
* and how to run it => but that was already mentioned

Reply:
To me, it boils down back to the familiarity of the user with maven. 
Since all operations you mentioned have their maven lifecycle (mvn install, mvn verify, mvn test)
Considering coverage, should we provide local coverage? We'll include the sonar badges that indicate our coverage , code smells, ... but I don't see why a developer wants to manually check coverage themselves? (at least not in a scenario where we need to assist him in this)

Reply:
I understand, but it doesn't take that much effort to add it so it is clear for QA people. The above is actually a min set of documentation that I add mostly in my README.md, how to build, how to test, how to dockerize.

With respect to the code review this is currently not supported by the SonarCloud that is used for this repository. Also the code coverage it not only about a number but actually checking which code parts are still not hit. 80% is a threshold defined number, but the test should mostly cover certain pieces of code. This you do check as a developer with manual checks imho. 

Maybe we have to discus with other stakeholders on this matter about the few mvn instructions to document for not experience java / mvn users.  Because in the end you might also say... read the code to understand this component :-)